packages/runtime/src/internal/Renderer.ts

```diff
@@ -639,6 +639,10 @@ async function legacyRenderBrick(
   let confProps: Record<string, unknown> | undefined;
   if (brickName === FORM_RENDERER) {
     ({ formData, ...confProps } = brickConf.properties ?? {});
+
+    if (brickConf.properties?.computed) {
```

Comment:
```suggestion
    if (brickConf.properties?.compute) {
```

这样吧，这个属性是表示数据还没被 compute、需要进行 compute，而不是已经被 computed

Reply:
done